react-toggle-switch-component/src/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import ReactDom from 'react-dom'
+
+class ToggleSwitch extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {isClicked: false}
+    this.flip = this.flip.bind(this)
+  }
+  flip() {
+    this.setState({
+      isClicked: !this.state.isClicked
+    })
+   }
+  render () {
+    let isClicked = this.state.isClicked
+    let position;
+    let positionClass;
+    let containerBG;
+
+    isClicked ? position = 'On' : position = 'Off'
```

Comment:
When using a turnery to set a variable it is better to use the following format:

```JSX
const position = isClicked ? 'On' : 'Off';
```